deploy.cmd

```diff
@@ -65,6 +65,13 @@ IF /I "%IN_PLACE_DEPLOYMENT%" NEQ "1" (
   IF !ERRORLEVEL! NEQ 0 goto error
 )
 
+:: 2. Check if WebConfig File exists
+IF EXIST %DEPLOYMENT_SOURCE%/web.config (
+  cp %DEPLOYMENT_SOURCE%/web.config %DEPLOYMENT_TARGET%/web.config
```

Comment:
Feel free to do it. I never used KuduSync, because that used plain old `cp` command.

Anyways, if KuduSync can check the file existence and copy it, awesome.